apps/studio/components/layouts/PageLayout/PageLayout.tsx

```diff
@@ -66,11 +68,12 @@ export const PageLayout = ({
   className,
   size = 'default',
   isCompact = false,
+  pageMeta,
```

Comment:
is this one being used? or it'll be used in future PRs?
we should add JSDocs for what does is too as I'm not sure what this is supposed to be in PageLayout